omniledger/service/service_test.go

```diff
@@ -296,3 +306,14 @@ func closeQueues(local *onet.LocalTest) {
 		close(services[0].(*Service).CloseQueues)
 	}
 }
+
+func registerDummy(services interface{}) {
```

Comment:
is it not possible to use the private method for tests?, i.e. `s.registerVerification`

Reply:
yes, that would also be possible - as well as writing the map directly in the service. But like this we also test `RegisterVerification`...